docs/PacketFence_Developers_Guide.asciidoc

```diff
@@ -57,6 +57,305 @@ To do so, the preferred way is using the `perldoc` command as follows:
   perldoc lib/pfconfig/cached.pm
 
 
+== AsciiDoc documentation
+
+=== Documentation Conventions
+
+Source: link:https://redhat-documentation.github.io/asciidoc-markup-conventions/[AsciiDoc Mark-up Quick Reference for Red Hat Documentation]
+
+[NOTE]
+====
+Some markup examples used in this overview are based on new Asciidoctor features and they are not a part of the standard set of elements. Please, use the `:experimental:` tag in the header of your document to enable this functionality.
+====
+
+[cols="3,4,4"]
+|===
+|Element|Mark-up|Example rendered output
+
+|Application name
+a|No special markup
+a| The foo application.
+
+|Code blocks
+a|
+....
+[source,java]
```

Comment:
I think we should use a Perl or Golang example. A Java example when there's no Java in PacketFence doesn't make sense.

Reply:
Done.